package/pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.openmrs.distro</groupId>
 	<artifactId>referenceapplication-package</artifactId>
-	<version>2.8.0-SNAPSHOT</version>
+	<version>2.8.0</version>
```

Comment:
Can you create a new pull request where you do not touch this file at all?

Reply:
Actually the current pull request i did exactly what you are suggesting

Reply:
@dkayiwa that may perhaps not build if other poms are having changed version or even the mismatch may be unbecoming in the tag, seems the conflict was with the change @jwnasambu had made on systemadmin owa version.
I have checked out her branch and rebuilt it until i zeroed to that area
 i see lots of module version changes as well, i hope these are ok?